### DISCUSSION TITLE
People not needing *accounts*? How does everyone do it?

### DISCUSSION THREAD

**Discussion by Lombra** [snipe/snipe-it#9306] [Q&A]
Hi,

So, I found this platform a while back and got intrigued. Our inventory management is awful. No track of who uses what, so this would be great.

But when I got to the user creation part I got a bit stumped. I need to come up with a username and a password for everyone. We'd be creating hundreds of users on dozens of companies (customers), none of which will ever login. To be honest it's kind of a deal breaker. I had planned to integrate the platform into our own using the API, so I could probably generate some random data that way, but still...

Anyway, I'm not only here to rant. I'm curious how people are using this platform that they want usernames for everyone. Is mine a particularly rare use case? I mean, maybe within your own organisation I can see that there's some merit to it.

**Comment by AlexanderWPapyrus** [comment]
Hi,
we auto sync our users with our active directory.
But if you want to use it for external users, I assume you already have some kind of database (or even Excel etc) with some unique ID if for every 'user' 
In this case you could import it into snipeit with customerID(or whatever) as username. Like in this official template:
[sample csv](https://snipeitapp.com/assets/downloads/sample-users.csv)
[documentation](https://snipe-it.readme.io/docs/importing)

btw a login for each user is not necessary.
But yeah, as snipe-it is focused to maintain only company internal assets and users, it's kinda a workaround.
Best wihses,
Alex

**Comment by csayre** [comment]
I would suggest using the active directory sync feature. Just a side note if you decide to go the AD route, make sure to set up an automated script to setup auto sync (this will automatically import new users as they are created). If not you can create users in the system by hand or by using an import.
